sources/packages/backend/apps/api/src/services/application-exception/application-exception.service.ts

```diff
@@ -57,19 +61,28 @@ export class ApplicationExceptionService extends RecordDataModelService<Applicat
       .innerJoin("exception.exceptionRequests", "exceptionRequest")
       .leftJoin("exception.exceptionNote", "exceptionNote")
       .leftJoin("exception.assessedBy", "assessedBy")
-      .where("exception.id = :exceptionId", { exceptionId })
-      .getOne();
+      .where("exception.id = :exceptionId", { exceptionId });
+
+    if (studentId) {
+      exception
+        .innerJoin("exception.application", "application")
+        .innerJoin("application.student", "student")
+        .andWhere("student.id = :studentId", { studentId });
+    }
+    return exception.getOne();
   }
 
   /**
    * Get a student application exception detected after the student application was
    * submitted, for instance, when there are documents to be reviewed.
    * @param applicationId application associated with the exception.
    * @param status statuses to be filtered.
+   * @param studentId student id.
```

Comment:
The order of comments studentId and status are reversed.

Reply:
I had no idea, that we were following the order in the comments.  I have seen comments without following the order.  did we decide it lately?

Reply:
We never enforced it on PR reviews but I would say that naturally they are usually ordered.
Is it good to have them ordered? IMO, yes.
When we create a new method, VS Code will do it for us and I believe that we should try to keep it, but not as a blocker.

Reply:
I agree with that, its good to have, when i suddenly i saw these comment, I thought we had an agreement on it and i might have the discussion. I have updated it